force-app/main/shared/classes/EventHandlerTest.cls

```diff
@@ -0,0 +1,99 @@
+@isTest
+private without sharing class EventHandlerTest {
+
+	@testSetup
+	private static void setup() {
+        TAG_TestDataFactory.assignPermissionSet( 'accountTypeHandler@nav.test', 'ManageAccounts' );
+        }
+    
```

Comment:
Lag også en ´testOnAfterInsert_200records´ testfunksjon for å teste om koden skaleres bra.